multisrc/overrides/mymangacms/manhwa18/src/Manhwa18.kt

```diff
@@ -0,0 +1,59 @@
+package eu.kanade.tachiyomi.extension.en.manhwa18
+
+import eu.kanade.tachiyomi.multisrc.mymangacms.MyMangaCMS
+import eu.kanade.tachiyomi.source.model.FilterList
+
+class Manhwa18 : MyMangaCMS("Manhwa18", "https://manhwa18.com", "en") {
+
```

Comment:
Bump the `versionId` so the ID will change and users will be forced to migrate to update the URLs.

```suggestion
class Manhwa18 : MyMangaCMS("Manhwa18", "https://manhwa18.com", "en") {

    // Migrated from FMReader to MyMangaCMS.
    override val versionId = 2

```